### ISSUE TITLE
Error when moving property types

### ISSUE THREAD

**Issue by w00kie-n00kie** [spopovic/uSiteBuilder#6]
Recieved an error when moving a property from a parent to a child. ie create a property in doc type and sync, then edit the code and move the property to a child doc type.

The error is in file : Vega.USiteBuilder.ManagerBase 
Will error on line 178 or 184 (depends upon whether there is a tab declared or not).

Code on lines 

178 - contentType.MovePropertyType(propertyType.Alias, propAttr.TabAsString);
184 - contentType.MovePropertyType(propertyType.Alias, null);

Basically the propertyType is null. Debugged the code and it looks like the PropertyType is not being correctly added to the ContentType (its listed in the CompositionPropertyTypes but not the PropertyTypes).
